pkgs/desktops/cinnamon/xapps/default.nix

```diff
@@ -0,0 +1,41 @@
+{ fetchFromGitHub, glib, gobjectIntrospection, gtk3, libgnomekbd, libxklavier, meson, ninja, pkgconfig, python3, python3Packages, stdenv, vala, wrapGAppsHook }:
+
+stdenv.mkDerivation rec {
+  pname = "xapps";
+  version = "1.6.2";
+
+  src = fetchFromGitHub {
+    owner = "linuxmint";
+    repo = "xapps";
+    rev = "${version}";
+    sha256 = "08fkypbx1j1kp3hp0y88idbj0la17hjwmw2w6f6jijwdipgwax7b";
+  };
+
+  patches = [
+    ./py-override.patch
```

Comment:
Could you add comment with a source and explanation?

Reply:
I believe it's a patch I wrote for @cawilliamson from https://github.com/linuxmint/xapps/pull/65.

Reply:
@worldofpeace What does the patch do (what should I add as comment)?

Reply:
I'd just add a comment with the link to the PR. It's all described there.